system-int-test/src/cmr/system_int_test/utils/search_util.clj

```diff
@@ -179,11 +179,12 @@
          url (url/retrieve-concept-url concept-type concept-id revision-id)
          url (if url-extension
                (str url "." url-extension)
-               url)]
-     (client/get url (merge {:accept (when-not url-extension format-mime-type)
-                             :throw-exceptions false
-                             :connection-manager (s/conn-mgr)}
-                            options)))))
+               url)
+         args (merge {:accept (when-not url-extension format-mime-type)
+                      :throw-exceptions false
+                      :connection-manager (s/conn-mgr)}
+                     options)]
```

Comment:
Note that this change was done so I could easily look at the args getting passed to the client (debugging different results from curl and CMR; first thing I wanted to check was that the args for the HTTP GET were the same). I left the change in place so that it would be easier for folks in the future, should they also want to debug the args.